cve_bin_tool/checkers/libebml.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+"""
+CVE checker for libebml
+
+https://www.cvedetails.com/product/33126/Matroska-Libebml.html?vendor_id=7864
+
+"""
+
+from cve_bin_tool.checkers import Checker
+
+
+class LibebmlChecker(Checker):
+    CONTAINS_PATTERNS = []
+    FILENAME_PATTERNS = [r"libebml.so"]
+    VERSION_PATTERNS = [
+        r"libebml-([0-9]+\.[0-9]+\.[0-9]+)",
+        r"([0-9]+\.[0-9]+\.[0-9]+)\nUnknown\nEBMLVoid",
```

Comment:
This is... a very strange version string.  I think it'll work but I just wanted to point it out because I suspect it may be a bit brittle.   Maybe we could put a comment in to that effect?

```suggestion
        r"([0-9]+\.[0-9]+\.[0-9]+)\nUnknown\nEBMLVoid",  # This string may be brittle to changes in string ordering
```